docker-compose.yml

```diff
@@ -3,16 +3,19 @@ services:
   dns:
     image: lancachenet/lancache-dns:latest
     env_file: .env
+    # restart: unless-stopped
```

Comment:
I think I would suggest ``#     restart` instead of `    # restart` otherwise you are removing a # and a space which will probably confuse non yml familiar people

Reply:
Sure, done.